src/System.Console/src/System/ConsoleCancelEventArgs.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System 
+{
+
+    public delegate void ConsoleCancelEventHandler(Object sender, ConsoleCancelEventArgs e);
+
+    public sealed class ConsoleCancelEventArgs : EventArgs
+    {
+        private ConsoleSpecialKey _type;
```

Comment:
Could be readnly.  Cancel could also be made an auto-prop, but either way is fine.